Examples/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-alert": 0
```

Comment:
FYI, you can do

```
/* eslint no-alert: 0 */
```

at the top of the file in order to disable it for a specific file